src/main/java/org/jabref/logic/importer/fetcher/SpringerFetcher.java

```diff
@@ -118,7 +119,7 @@ public static BibEntry parseSpringerJSONtoBibtex(JSONObject springerJsonEntry) {
                 urls.forEach(data -> {
                     JSONObject url = (JSONObject) data;
                     if (url.optString("format").equalsIgnoreCase("pdf")) {
-                        entry.addFile(new LinkedFile("online", url.optString("value"), "PDF"));
+                        entry.addFile(new LinkedFile("online", Path.of(url.optString("value")), "PDF"));
```

Comment:
I doubt this change will work. You are trying to resolve a http url as local path object.
> The Path is obtained by invoking the getPath method of the default FileSystem. 
https://docs.oracle.com/en/java/javase/13/docs/api/java.base/java/nio/file/Path.html#of(java.lang.String,java.lang.String...)

Reply:
Should be fixed.